app/controllers/organisation_relationships_controller.rb

```diff
@@ -14,7 +14,7 @@ class OrganisationRelationshipsController < ApplicationController
   ]
 
   def stock_owners
-    stock_owners = organisation.stock_owners.filter_by_active
+    stock_owners = organisation.stock_owners.filter_by_active.order('LOWER(name)')
```

Comment:
With just .order the sort was case-sensitive and we get incorrect ordering e.g. MHCLG before Managing Agent 1. Using LOWER(name) ensures the sorting of names is case-insensitive.